test/unit/events.js

```diff
@@ -66,4 +66,52 @@ describe('custom events', () => {
       done();
     }, 1);
   });
+
+  it('should handle cases correctly', done => {
```

Comment:
Maybe a more descriptive name as to what exactly it's testing? Maybe something like "should preserve declared casing".